example/integration_test/map_interface_test.dart

```diff
@@ -200,11 +200,7 @@ void main() {
 
     var options = await mapboxMap.getResourceOptions();
     expect(options.accessToken, isNotNull);
-    if (Platform.isAndroid) {
-      expect(options.baseURL, 'https://api.mapbox.com');
-    } else {
-      expect(options.baseURL, 'file:///https:/api.mapbox.com');
```

Comment:
what was the reason for the file prefix?

Reply:
This was caused by a bug in the iOS SDK https://github.com/mapbox/mapbox-maps-ios/pull/1749, which was fixed in 10.11.0.